CHANGELOG.md

```diff
@@ -25,7 +25,7 @@
 * BC BREAK - Fix Mockery not trying default expectations if there is any concrete expectation
 * BC BREAK - Mockery's PHPUnit integration will mark a test as risky if it
   thinks one it's exceptions has been swallowed in PHPUnit > 5.7.6. Use `$e->dismiss()` to dismiss.
-
+* Added possibility to add Constructor-Expections on hard dependancies, read: Mockery::mock('overload:...')
```

Comment:
*dependancies* isn't a word. Applies throughout this PR.

Reply:
Please provide a suitable replacement.

At least my search bubble yielded some interesting results:
https://en.oxforddictionaries.com/definition/dependency
> ‘the islands became dependencies of Norway and Denmark’
https://askubuntu.com/questions/361741/what-are-dependencies
> By dependencies are missing the package manager...

hth

Reply:
@Bilge means it is spelt incorrectly, you have the right word. It has 3 Es, 0 As.

Reply:
lol now i see it, thanks @davedevelopment